pyaerocom/config.py

```diff
@@ -310,15 +291,19 @@ def _infer_config_from_basedir(self, basedir):
         )
 
     def infer_basedir_and_config(self):
-        """Boolean specifying whether the lustre database can be accessed"""
-        for sub_envdir, cfg_id in self._DB_SEARCH_SUBDIRS.items():
-            for sdir in self._basedirs_search_db():
-                basedir = os.path.join(sdir, sub_envdir)
-                if self._check_access(basedir):
-                    _chk_dir = os.path.join(basedir, self._check_subdirs_cfg[cfg_id])
-                    if self._check_access(_chk_dir):
-                        return (basedir, self._config_files[cfg_id])
-        raise FileNotFoundError("Could not establish access to any registered database")
+        """
+        check if ~/MyPyaerocom/paths.ini exists.
+        if not, use the default paths.ini
+        """
+
+        self._paths_ini = os.path.join(self.my_pyaerocom_dir, self.PATHS_INI_NAME)
+        if os.path.exists(self._paths_ini):
+            logger.info(f"using user specific config file: {self._paths_ini}")
+        else:
+            with resources.path("pyaerocom.data", self.PATHS_INI_NAME) as path:
```

Comment:
Maybe write a log message in this case as well. Could be useful to debug cases where eg. user config is not picked up due to typos in path.